roofit/histfactory/src/MakeModelAndMeasurementsFast.cxx

```diff
@@ -374,12 +374,7 @@ void RooStats::HistFactory::FitModelAndPlot(const std::string& MeasurementName,
   }
 
   // Loop over all POIs and print their fitted values
-  RooRealVar* poi = NULL; // (RooRealVar*) POIs->first();
-  TIter params_itr = POIs->createIterator();
-  TObject* poi_obj=NULL;
-  while( (poi_obj=params_itr.Next()) ) {
-    //poi = (RooRealVar*) poi_obj;
-    poi = dynamic_cast<RooRealVar*>(poi_obj);
+  for (auto const *poi : dynamic_range_cast<RooRealVar *>(*POIs)) {
```

Comment:
```suggestion
  for (auto const *poi : static_range_cast<RooRealVar *>(*POIs)) {
```
We can use static cast here because the output is never checked for `nullptr`.